com.civicrm.hrjobroles/CRM/Hrjobroles/BAO/Query.php

```diff
@@ -90,6 +100,18 @@ private function isAJobRoleParam($param) {
         return false;
     }
 
+  /**
+   * Applies date filtering for active job roles
+   *
+   * @param CRM_Contact_BAO_Query $query
+   */
+    private function applyJobRoleDateFiltering(&$query) {
```

Comment:
Please check if there are existing tests for this class, if so we need to add for this change too.